README.md

```diff
@@ -78,7 +78,7 @@ If you wish to build Memray from source you need the following binary dependenci
 - libunwind (for Linux)
 - liblz4
 
-Check your package manager on how to install these dependencies (for example `apt-get install libunwind-dev liblz4-dev` in Debian-based systems
+Check your package manager on how to install these dependencies (for example `apt-get install build-essential python3.12-dev libunwind-dev liblz4-dev` in Debian-based systems
```

Comment:
Let's not pin this to a python version
```suggestion
Check your package manager on how to install these dependencies (for example `apt-get install build-essential python3-dev libunwind-dev liblz4-dev` in Debian-based systems
```

on a debian system `python3-dev` should install the right headers